README.md

```diff
@@ -1,334 +1,27 @@
 # Jupyter Kernel Gateway
 
-A web server that supports different mechanisms for spawning and communicating with Jupyter kernels, such as:
+A web server that supports different mechanisms for spawning and
+communicating with Jupyter kernels, such as:
 
-* A Jupyter Notebook server-compatible HTTP API for requesting kernels and talking the [Jupyter kernel protocol](http://jupyter-client.readthedocs.org/en/latest/messaging.html) with them over Websockets
-* A HTTP API defined by annotated notebook cells that maps HTTP verbs and resources to code to execute on a kernel
+* A Jupyter Notebook server-compatible HTTP API used for requesting kernels
+  and talking the [Jupyter kernel protocol](http://jupyter-client.readthedocs.org/en/latest/messaging.html)
+  with the kernels over Websockets
+* A HTTP API defined by annotated notebook cells that maps HTTP verbs and
+  resources to code to execute on a kernel
 
-The server launches kernels in its local process/filesystem space. It can be containerized and scaled out by a cluster manager (e.g., [tmpnb](https://github.com/jupyter/tmpnb)).
+The server launches kernels in the server's local process/filesystem space.
+The server can be containerized and scaled out by a cluster manager (e.g.,
+[tmpnb](https://github.com/jupyter/tmpnb)).
 
```

Comment:
When the doc is posted, we should add a direct link to it here "See the features page in the documentation for more details."